client/lib/plans/features-list.js

```diff
@@ -1315,6 +1329,67 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'Live chat support 24X7' ),
 		getDescription: () => i18n.translate( 'Live chat is available 24/7.' ),
 	},
+
+	/* START - Jetpack tiered product-specific features */
+	[ FEATURE_TIERED_REAL_TIME_BACKUPS ]: {
+		getSlug: () => FEATURE_TIERED_REAL_TIME_BACKUPS,
+		getTitle: () =>
+			i18n.translate( 'Real-time backups {{noHighlight}}(as you edit){{/noHighlight}}', {
+				components: {
+					/* eslint-disable-next-line wpcalypso/jsx-classname-namespace */
+					noHighlight: <span className="not-highlighted" />,
+				},
+			} ),
+		isHighlighted: () => true,
+	},
+	[ FEATURE_TIERED_CLOUD_BASED_BACKUPS ]: {
+		getSlug: () => FEATURE_TIERED_CLOUD_BASED_BACKUPS,
+		getTitle: () => i18n.translate( 'Backups are cloud based' ),
+	},
+	[ FEATURE_TIERED_20GB_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_TIERED_20GB_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( '20GB of backup storage' ),
+	},
+	[ FEATURE_TIERED_1TB_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_TIERED_1TB_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( '1TB of backup storage' ),
+	},
+	[ FEATURE_TIERED_UNLIMITED_BACKUP_STORAGE ]: {
+		getSlug: () => FEATURE_TIERED_UNLIMITED_BACKUP_STORAGE,
+		getTitle: () => i18n.translate( 'Unlimited backup storage' ),
+	},
+	[ FEATURE_TIERED_ONE_CLICK_RESTORES_30_DAYS ]: {
+		getSlug: () => FEATURE_TIERED_ONE_CLICK_RESTORES_30_DAYS,
+		getTitle: () => i18n.translate( 'One-click restore from last 30 days' ),
+	},
+	[ FEATURE_TIERED_ONE_CLICK_RESTORES_1_YEAR ]: {
+		getSlug: () => FEATURE_TIERED_ONE_CLICK_RESTORES_1_YEAR,
+		getTitle: () => i18n.translate( 'One-click restore from the last year' ),
+	},
+	[ FEATURE_TIERED_ONE_CLICK_RESTORES_UNLIMITED ]: {
+		getSlug: () => FEATURE_TIERED_ONE_CLICK_RESTORES_UNLIMITED,
+		getTitle: () => i18n.translate( 'One-click restores from any time' ),
+	},
+	[ FEATURE_TIERED_PRODUCT_BACKUP ]: {
+		getSlug: () => FEATURE_TIERED_PRODUCT_BACKUP,
+		getTitle: () => i18n.translate( 'All Backup features' ),
+		isHighlighted: () => true,
+	},
+	[ FEATURE_TIERED_PRODUCT_SCAN ]: {
+		getSlug: () => FEATURE_TIERED_PRODUCT_SCAN,
+		getTitle: () => i18n.translate( 'Scan (real-time & automated)' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Scan (real-time, automated)' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/tiered-product-cards-copy)